src/plugin.ts

```diff
@@ -76,18 +82,22 @@ export function openGraphImagePlugin(options: Options): Plugin {
   const appUrlPromise = new DeferredPromise<URL>()
   const routesWithImages = new Set<ConfigRoute>()
 
-  async function fromRoot(...paths: Array<string>): Promise<string> {
-    const viteConfig = await viteConfigPromise
-    return path.resolve(viteConfig.root, ...paths)
-  }
-
   async function fromRemixApp(...paths: Array<string>): Promise<string> {
     const remixContext = await remixContextPromise
     return path.resolve(remixContext.remixConfig.appDirectory, ...paths)
   }
 
+  async function fromViteBuild(...paths: Array<string>): Promise<string> {
+    const remixContext = await remixContextPromise
+    return path.resolve(
+      remixContext.remixConfig.buildDirectory,
+      'client',
+      ...paths
+    )
+  }
+
   async function fromOutputDirectory(...paths: Array<string>): Promise<string> {
-    return fromRoot(options.outputDirectory, ...paths)
+    return fromViteBuild(options.outputDirectory, ...paths)
```

Comment:
this generates the build directly to the `/client` directory. will it make sense to have a copy in `/public` so that it can be used when using dev server? i don't really see a usecase and directly having them under `/client` should be fine. what do you think?

Reply:
I'd discuss this separately. I'd say there's no need to copy it to `public`. Since the plugin is build-only, you are likely to observe an _oudated_ image in `public`, which isn't nice.